ltr_tgs.txt

```diff
@@ -8,35 +8,27 @@
 |- align="center"
 |0-01-028||Waltham||Perfection Towing||2018.03.14 OceanNora
 |- align="center"
-|1-01-030||Hudson||Dee Bus Service - North Middlesex and Lunenburg School Bus Ops ||2022.01.03 MASSBUSMARK
+|0-01-030||Hudson||Dee Bus Service - North Middlesex and Lunenburg School Bus Ops ||2023.01.06 N1ZYY
 |- align="center"
 |0-09-030||Georgetown||Unknown Chair Car Service - Dispatch to Mobiles Only||2018.10.13 GaryS
 |-align="center"
 |0-09-031||Georgetown||Unknown Chair Car Service - Mobiles to Dispatch only||2018.10.13 GaryS
 |-align="center"
-|0-01-032||Waltham||Unknown School Bus Service||2022.01.03 MASSBUSMARK
+|0-01-032||Waltham||Unknown School Bus Service||2023.01.06 MASSBUSMARK
```

Comment:
Simple reconf.

I actually have an inkling for where they are operating, but a troubling amount of radio traffic was effectively, "I'm at [presumably a student's address] and their parents aren't home" sort of stuff. As such, I don't feel great about trying to more precisely identify this bus company for publication.